app/services/send_application_to_provider.rb

```diff
@@ -0,0 +1,26 @@
+# This worker will be scheduled to run nightly
+class SendApplicationToProvider
+  attr_accessor :application_choice
+
+  def initialize(application_choice:)
+    self.application_choice = application_choice
+  end
+
+  def call
+    set_reject_by_default
+    ApplicationStateChange.new(application_choice).send_to_provider!
+  end
+
+private
+
+  def set_reject_by_default
+    days = TimeLimitCalculator.new(
+      rule: :reject_by_default,
+      effective_date: Time.zone.now,
+    ).call
+    return unless days
```

Comment:
Can this ever be nil?

Reply:
Well I guess not any more. When the config was in the DB this made sense but not now - will remove.